src/PrestaShopBundle/Form/Admin/Sell/Product/ProductFormType.php

```diff
@@ -80,6 +80,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('basic', BasicType::class, [
                 'product_id' => $productId,
             ])
+            ->add('description', DescriptionType::class, [
```

Comment:
I don't think you need this new type, you should move the categories type from the Shortcut type into the Basic one (we will rename it into DescriptionType in the PR that handles the split of tabs)